.gitignore

```diff
@@ -1,3 +1,6 @@
 # Files created during packaging
 .manifest.js.bak
 .dist
+
+# vscode files
+.vscode
```

Comment:
does this belong here

Reply:
just for the vscode tasks.json (I have a automated pull from upstream). should I remove it?